### ISSUE TITLE
Exporting 2 cubes with UV maps + draco compression => WebGL Error

### ISSUE THREAD

**Issue by donalffons** [KhronosGroup/glTF-Blender-IO#917]
Hi,

it took me more than a day of trying to narrow this weird problem down. Attached is a sample file with two cubes and a texture applied:
![image](https://user-images.githubusercontent.com/33040347/74091695-75dcd280-4aba-11ea-91a6-073a6d3d70a3.png)

If I export this scene to GLTF / GLB and open it up with https://gltf-viewer.donmccurdy.com/, it shows the following errors:
![image](https://user-images.githubusercontent.com/33040347/74091706-9dcc3600-4aba-11ea-8013-df20ad69a7ba.png)

Visually, everything seems OK. However, I have a large model which gives the same error message and misses 90% of the objects when I load it in ThreeJS. I assume that these issues are connected.

The error is somehow related to the UV maps and DRACO compression. If I remove the UV map on one of the cubes, it solves the issue. If I disable DRACO compression, it also solves it.

[gltf bug report.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/4239566/gltf.bug.report.zip)
**Update 2020-02-22: Fixed missing texture in blend file**

I would greatly appreciate any help with this issue.

**Version**
 - OS: Manjaro Linux
 - Blender Version: 2.83 / 2.82

**Comment by julienduroure** [comment]
@McNopper Can you please have a look?

**Comment by UX3D-nopper** [comment]
My team will have a look at it.

**Comment by donalffons** [comment]
Hi,
I have a few updates on the issue

- I tested the file on Windows with today's Blender 2.83 version and it shows similar errors (weird thing is that the objects don't even show). Below picture shows the console errors from the windows export.
![image (3)](https://user-images.githubusercontent.com/33040347/75095331-e7755000-5593-11ea-9772-766de29247da.png)

- I tested with a couple of different blender version I have lying around on my (Manjaro Linux) computer. The latest Blender version I have, which produces no error is
blender-2.82-3ed7ff1f9f60-linux-glibc217-x86_64 (Sat Nov 9 10:05:30 2019 +1100)
After that commit, the next version I have which produces the error is
blender-2.82-b03066f7ee07-linux-glibc217-x86_64 (Wed Dec 11 19:51:56 2019 +0100)
I would assume that somewhere between those two commits, the problem must have been introduced...
